edb/server/protocol/ai_ext.py

```diff
@@ -2225,6 +2232,9 @@ async def _handle_embeddings_request(
             raise TypeError(
                 'missing or empty required "model" value in request')
 
+        shortening = body.get("dimensions")
```

Comment:
I'd like above on line 2225 to use ` inputs = body.get("inputs")` and to expect `inputs` from the clients, but this is a breaking change so probably we wanna leave it as `input`.

Reply:
Sure, accept both, but throw an error when both are specified.